Source/diablo.cpp

```diff
@@ -926,10 +926,12 @@ void DiabloInit()
 	SetApplicationVersions();
 
 	for (size_t i = 0; i < QUICK_MESSAGE_OPTIONS; i++) {
-		if (strlen(sgOptions.Chat.szHotKeyMsgs[i]) != 0) {
+		auto &messages = sgOptions.Chat.szHotKeyMsgs[i];
+		if (messages.size() > 1 || messages[0].length() != 0) {

```

Comment:
Quick messages are a little bit special, cause we can't set the default message when loading the `ini` options.
The reason is that the default depends on the language and the language is initialized **after** the `ini` options are loaded.

To handle this I [changed the logic](https://github.com/diasurgical/devilutionX/pull/3241#discussion_r762578885) that we don't populate the vector if we don't have an entry in the `ini` file.
This allows the logic in `diablo.cpp` to check if something is present in the `ini` (`size() > 0`). If not set the default value.

The only other thing is that we need to always create a default entry in the `ini` so that we don't run the default initialize multiple times (done in `SetIniValue`). This is similar to all other `ini` setters. We always create an entry, even if it's a empty default.